docs/en/setup/backend/backend-infrastructure-monitoring.md

```diff
@@ -0,0 +1,115 @@
+# VMs monitoring 
+SkyWalking leverages Prometheus node-exporter for collecting metrics data from the VMs, and leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](backend-receivers.md#opentelemetry-receiver) and into the [Meter System](./../../concepts-and-designs/meter.md).  
+We defined the VM entity as a `Service` in OAP, use `vm::` as a prefix to identify.  
+
+## Data flow
+1. Prometheus node-exporter collects metrics data from the VMs.
+2. OpenTelemetry Collector fetches metrics from node-exporter via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus GRPC Exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results. 
+
+## Setup 
+1. Setup [Prometheus node-exporter](https://prometheus.io/docs/guides/node-exporter/).
+2. Setup [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/). This is an example for OpenTelemetry Collector configuration [otel-collector-config.yaml](../../../../test/e2e/e2e-test/docker/promOtelVM/otel-collector-config.yaml).
```

Comment:
Why k8s monitoring has own config file in the doc, and this doesn't?

Reply:
VMs and K8s monitoring's `otel-collector-config.yaml` are different, I reused the VMs configuration which in E2E